src/app/navigation-carousel/navigation-carousel.component.css

```diff
@@ -0,0 +1,30 @@
+.navigation {
+  position: fixed;
+  left: 12px;
+  top: 210px;
+}
+
+ol {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  border: 2px solid white;
+  border-radius: 100%;
+  padding: 4px;
+  transition: all 0.25s ease;
+  cursor: pointer;
+  margin-top: 36px;
+}
+
```

Comment:
You don't need it. `li:hover {
    background-color: white;
} ` 
Because color will just changed, once the navigation is done.